client/src/App.js

```diff
@@ -24,16 +25,50 @@ const theme = createMuiTheme({
 });
 
 function App() {
-  const [authStatus, setAuthStatus] = useState(true);
+  const [authStatus, setAuthStatus] = useState(false);
```

Comment:
So, first, this controls which Nav bar shows up, so there is one of two options here:
1. When you're setting user's state, you also set the state of authStatus.
2. You change the boolean logic to just use the user's state to see if it is null or not.